usr/share/rear/init/default/100_check_stale_nfs_mounts.sh

```diff
@@ -0,0 +1,11 @@
+# 100_check_stale_nfs_mounts.sh
+# Purpose is to have a simple test if we have stale NFS mount points which
+# could lead to a hangind ReaR session.
+# In case there is a stale NFS present we bail out with an error to get it fixed before
+# running ReaR again.
+# See issue https://github.com/rear/rear/issues/3109
+cat /proc/mounts | grep -i "[[:blank:]]nfs" | \
```

Comment:
Maybe only style: I'd put the `cat` and `grep` in a subshell via `< <()` to keep the `while` loop at the top level of the Bash. This also helps with the variable scoping in case one would want to modify variables defined outside the loop.

Reply:
Nice - done it as suggested.